src/Wt/WTable.C

```diff
@@ -451,6 +458,16 @@ void WTable::moveColumn(int from, int to)
   repaint(RepaintFlag::SizeAffected);
 }
 
+void WTable::updateRowNumbers(int begin, int end) {
+  if (begin < 0 || begin >= rowCount() || end < 0 || end >= rowCount()) {
+    return;
```

Comment:
If this ever happens that would be a programming error that's internal to Wt, right?

This should then either cause an alarming error message or should be omitted altogether.